app/views/users/show.html.erb

```diff
@@ -100,6 +100,14 @@
     <div style="clear: both;"></div>
   <% end %>
 
+  <% if @user.is_admin? && @user.show_email? %>
```

Comment:
Hi @pushcx, could this be the check you are referring to?

Reply:
Also, I am up for suggestions on what text to use on the label.

Reply:
I was referring to the setting page. This `if` should be `||` so admins always see emails.